ui/react-components/components/ListView/ListView.spec.jsx

```diff
@@ -0,0 +1,142 @@
+    const tree = renderer
+        .create(<ListView />)
+        .toJSON()
+
+    expect(tree).toMatchSnapshot()
+})
+
+const columns = {
+    "Patient ID": "",
+    "Patient Name": "mid",
+    "Date": "",
+    "Start Time": "",
+    "End Time": "",
+    "Provider": "mid",
+    "Speciality": "",
+    "Service": "mid",
+    "Service Appointment Type": "mid",
+    "Status": "",
+    "Walk In": "",
+    "Location": "mid",
+    "Additional Info": "mid",
+    "Notes": ""
+}
+
+const rows = [
+    {
+        "Patient ID": "GAN203012",
+        "Patient Name": "Test Diabetes",
+        "Date": "24 Mar 20",
+        "Start Time": "10:00 am",
+        "End Time": "10:30 am",
+        "Provider": "",
+        "Speciality": "Cardiology",
+        "Service": "Cardiology",
+        "Service Appointment Type": "",
+        "Status": "Scheduled",
+        "Walk In": "Yes",
+        "Location": "OPD-1",
+        "Additional Info": "",
+        "Notes": "",
+    },
+    {
+        "Patient ID": "GAN203011",
+        "Patient Name": "Test Con",
+        "Date": "24 Mar 20",
+        "Start Time": "9:00 am",
+        "End Time": "9:30 am",
+        "Provider": "",
+        "Speciality": "Cardiology",
+        "Service": "Cardiology",
+        "Service Appointment Type": "",
+        "Status": "Scheduled",
+        "Walk In": "Yes",
+        "Location": "OPD-1",
+        "Additional Info": "",
+        "Notes": "Urgent",
+    },
+    {
+        "Patient ID": "GAN203019",
+        "Patient Name": "Test Con",
+        "Date": "22 Mar 20",
+        "Start Time": "11:00 am",
+        "End Time": "11:30 am",
+        "Provider": "",
+        "Speciality": "Psychology",
+        "Service": "Psychology",
+        "Service Appointment Type": "",
+        "Status": "Scheduled",
+        "Walk In": "Yes",
+        "Location": "General Ward",
+        "Additional Info": "",
+        "Notes": "Nothing important",
+    }
+]
+
+
+test('renders ListView with appointments correctly', () => {
+    const tree = renderer
+        .create(<ListView columns={columns} rows={rows} />)
+        .toJSON()
+
+    expect(tree).toMatchSnapshot()
+})
+
+test('should sort records ascending wrt column and display icon on click of column header', () => {
+    rows[0]["Patient Name"] = "Antony"
+    rows[1]["Patient Name"] = "Zenia"
+    rows[2]["Patient Name"] = "Ramesh"
+    const { getByText, container } = render(<ListView columns={columns} rows={rows} />)
+
+    fireEvent.click(getByText('Patient Name'))
+
+    expect(getByText("Patient Name").childNodes[1]).toHaveClass("fa fa-caret-up")
+
+    const tableBody = container.querySelector("tbody")
+    expect(tableBody.children[0].childNodes[1].textContent).toBe("Antony")
+    expect(tableBody.children[1].childNodes[1].textContent).toBe("Ramesh")
+    expect(tableBody.children[2].childNodes[1].textContent).toBe("Zenia")
+})
+
+test('should sort records descending wrt column and display icon on click of column header', () => {
```

Comment:
can we please stick to the current way of structuring tests ie. "describe" with "it" and not introduce new structures?